test_pull_requests

```diff
@@ -124,12 +124,37 @@ NEEDS_REBASE_LABEL          = "needs-rebase"
 GITHUB_API_BASE_URL = 'https://api.github.com'
 GITHUB_BASE_URL = 'https://github.com'
 
+# TODO: create a :state to "content" mapping for composing new bot
+# messages
+PR_STATES = {
+  /^Evaluating/ => :evaluating,
```

Comment:
> As far as not being states for the same thing, do you mean the `ACTION_*`
keys don't belong with the others? I'll try to find a way to break those
out.


I do.  I haven't stared at all the values.  But basically all the states in a map need to be about the same thing.